build.sbt

```diff
@@ -29,7 +29,7 @@ lazy val root = project.withId("spark-snowflake").in(file("."))
   .settings(
     name := "spark-snowflake",
     organization := "net.snowflake",
-    version := "2.5.9-spark_2.4",
+    version := "2.6.0-spark_2.4",
```

Comment:
Is this enabling Arrow by default? Or should we use 2.5.10 if we're not?

Reply:
This is only version change. It doesn't enable arrow. To enable arrow format, what we should do is to change the default value of 'use_copy_unload' as false.